lib/spack/spack/relocate.py

```diff
@@ -76,32 +77,16 @@ def __init__(self, old_path, new_path):
 
 
 def _patchelf():
-    """Return the full path to the patchelf binary, if available, else None.
-
-    Search first the current PATH for patchelf. If not found, try to look
-    if the default patchelf spec is installed and if not install it.
-
-    Return None on Darwin or if patchelf cannot be found.
-    """
-    # Check if patchelf is already in the PATH
-    patchelf = executable.which('patchelf')
-    if patchelf is not None:
-        return patchelf.path
-
-    # Check if patchelf spec is installed
-    spec = spack.spec.Spec('patchelf')
-    spec._old_concretize(deprecation_warning=False)
-    exe_path = os.path.join(spec.prefix.bin, "patchelf")
-    if spec.package.installed and os.path.exists(exe_path):
-        return exe_path
-
-    # Skip darwin
+    """Return the full path to the patchelf binary, if available, else None."""
     if is_macos:
         return None
 
-    # Install the spec and return its path
-    spec.package.do_install()
-    return exe_path if os.path.exists(exe_path) else None
+    patchelf = executable.which('patchelf')
```

Comment:
Can we do a follow-up PR where executable.which is replaced with getting a spec out of it through external find logic? Cause what we should require a lower-bound `@0.13:` on the version number, given that ubuntu:18.04 ships a very buggy patchelf 0.9.